app/src/main/java/de/westnordost/streetcomplete/quests/way_lit/AddWayLit.java

```diff
@@ -0,0 +1,66 @@
+package de.westnordost.streetcomplete.quests.way_lit;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import java.util.Arrays;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddWayLit extends SimpleOverpassQuestType
+{
+	static final String[] ROADS_WITH_LIGHT = {
+			"primary", "secondary", "tertiary", "unclassified", "residential", "living_street",
+			"service"
+	};
+
+	static final String[] WAYS_WITH_LIGHT = {
+			"pedestrian", "footway", "track", "road", "cycleway", "path"
```

Comment:
Oops, was "track" there before? If it was, I apologize: This should of course also be removed, see [wiki entry](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrack). It will have too many false positives out of town.

Also, I'd move "pedestrian" over to streets because pedestrian streets are usually still referred to as streets (because they are, with housenumbers and everything).